src/faker/provider/ssn/ssn_en_US.nim

```diff
@@ -0,0 +1,17 @@
+include interfaces
+
+proc areaNumber(f: Faker): int =
+  var areaNumber = f.rand.rand(001..899)
+  if areaNumber == 666:
+    (if f.rand.rand(0..1) == 1: inc(areaNumber) else: dec(areaNumber))
+  return areaNumber
+
+proc groupNumber(f: Faker): int =
+  return f.rand.rand(01..99)
```

Comment:
Need zero-padding.

```nim
import random

randomize()
echo rand(01..10)
# -> 8 (NG)
# -> 08 (OK)
```

Reply:
SSN format is:

* AAA-GG-SSSS

e.g. `001-02-0003`

Reply:
Since I'm new to Nim. Would it be a good idea to use strformat to add padding? I'm not sure of the preferred way for this.

Reply:
You can `strformat`

```nim
import random
import strformat

randomize()
let x = rand(001..100)
echo fmt"{x:>03}"
# -> 008
```

Reply:
Updated with zero padding